WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1468,15 +1468,18 @@ TaskInfo                = @ContactInfo@;
 
 # Task List view
 TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right:12px;
+TaskListViewNewTaskInputWidget                              = z-index:300; overflow:hidden; padding:0px 0px 0px 14px; border-width:0px;
```

Comment:
It was earlier used in the class, InlineWidget . 
I moved them to properties file